.changeset/cuddly-pumas-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': minor
+---
+
+Add hover state to disabled `.iui-field` and `.iui-list-item` to help with accessibility.
```

Comment:
1. Changesets should be worded in past tense.
2. CSS/variables changesets (especially for visual changes) should have corresponding React changesets describing the high-level effects of the underlying changes.

Reply:
Also can we enhance the documentation in some places to describe the outcome (and the rationale) of this PR? Developers should both understand the default behavior and be guided to do the right thing, to prevent end-user frustration ("why is this thing disabled?").

Reply:
1. Updated to past tense.
2. Some day I'll remember this. 😅 Added React changeset.
3. Updated variables documentation to list the new variables with a very broad explanation of why. Also updated PR description.

Reply:
The React changesets should not mention implementation details (i.e. CSS classes). As mentioned above, they should describe "the high-level effects of the underlying changes" (i.e. what changed visually and in which components).

Reply:
Updated React changeset to:
> Added hover state to disabled button, select, input, textarea, and actionable list item to help with accessibility.